docs/usage/nuget.md

```diff
@@ -69,6 +69,10 @@ All feeds are checked for dependency updates, and duplicate updates are merged i
     If your project has lockfile(s), for example a `package.lock.json` file, then you must set alternate feed settings in the `NuGet.config` file only.
     `registryUrls` set in other files are **not** passed to the NuGet commands.
 
+<!-- prettier-ignore -->
+!!! note
+    Some alternative feeds (e.g. Artifactory) do not implement the full set of [required NuGet resources](https://learn.microsoft.com/en-us/nuget/api/overview#resources-and-schema) for the V3 API. If the `PackageBaseAddress` resource does not exist, Renovate falls back to using the `projectUrl` from the catalog entry.
```

Comment:
Another sentence explaining what this "really" means to users would be best. Eg is this about source url and changelogs only?